Dockerfile

```diff
@@ -1,39 +1,34 @@
-FROM ubuntu:14.04
-
+FROM ubuntu:16:04
 MAINTAINER Kyle Dove <dovek@umich.edu>
```

Comment:
Probably shouldn't be a specific person since sometimes they move on to "better" things.